.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,190 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Build"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  coding-standards:
+    name: "coding standards"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.1"
+          - "7.2"
+          - "7.3"
+          - "7.4"
```

Comment:
Question is: do we need run this 

- against all PHP versions
- only the lowest supported one

?

I believe the latter suffices.